### ISSUE TITLE
Octave dev install from URL 'file_idx' undefined 

### ISSUE THREAD

**Issue by cbm755** [apjanke/octave-testify#6]
Octave dev tip:
```
octave:1> pkg install https://github.com/apjanke/octave-testify/releases/download/v0.1.0/testify-0.1.0.tar.gz
error: 'file_idx' undefined near line 471 column 21
error: called from
    pkg at line 471 column 31
```

**Comment by mtmiller** [comment]
Same, reported upstream [#55843](https://savannah.gnu.org/bugs/?55843).

**Comment by cbm755** [comment]
Fixed upstream, and works in released 5.1.0 so probably nothing required here, closing

**Comment by apjanke** [comment]
Cool. Noting as upstream issue.
